Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -284,6 +289,30 @@ await UpdateIntegrationRows(false, token)
                     .ConfigureAwait(false);
             }
         }
+        
+        /// <summary>
+        /// Task that runs in a loop, checking for new config in extraction pipelines
+        /// </summary>
+        public async Task CheckForNewConfig(CancellationToken token)
+        {
+            while (!token.IsCancellationRequested)
+            {
+                await Task
+                    .Delay(900000, token) // Run every 15 minutes
```

Comment:
use constant variable that is 9000

Reply:
check if you have specified u want local config, and this config runs, what will happen

Reply:
Ok its fine